tutoraspects/templates/aspects/jobs/init/clickhouse/init-clickhouse.sh

```diff
@@ -335,7 +335,7 @@ CREATE TABLE IF NOT EXISTS {{ ASPECTS_EVENT_SINK_DATABASE }}.{{ ASPECTS_EVENT_SI
 
 -- Grant permissions to the users
 GRANT INSERT, SELECT ON {{ ASPECTS_XAPI_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_LRS_USER }}';
-GRANT SELECT ON {{ ASPECTS_XAPI_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_REPORT_USER }}';
+GRANT SELECT, DROP TABLE, DROP VIEW ON {{ ASPECTS_XAPI_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_REPORT_USER }}';
```

Comment:
Non-select perms here can go away once @SoryRawyer creates a reporting schema for dbt to work in.